src/components/provider-list-item/provider-list-item.js

```diff
@@ -11,68 +11,84 @@ import styles from './provider-list-item.css';
 
 const cx = classNames.bind(styles);
 
-function ProviderListItem({ item, link, active, onClick, headingLevel }) {
-  return !item ? (
-    <div className={styles.skeleton} />
-  ) : (
-    <InternalLink
-      data-test-eholdings-provider-list-item
-      to={link}
-      className={cx('item', {
-        'is-selected': active
-      })}
-      onClick={(e) => {
-        if (onClick) {
-          e.preventDefault();
-          onClick();
-        }
-      }}
-    >
-      <Headline
-        data-test-eholdings-provider-list-item-name
-        margin="none"
-        size="medium"
-        tag={headingLevel || 'h3'}
-      >
-        {item.name}
-      </Headline>
-
+const ProviderListItem = ({
+  item,
+  link,
+  active,
+  onClick,
+  headingLevel,
+}) => {
+  return !item
+    ? (
       <div
-        data-test-eholdings-provider-list-item-selections
-        className={cx('itemMetadata')}
+        className={styles.skeleton}
+        data-testid="skeleton-element"
+      />
+    )
+    : (
+      <InternalLink
+        data-test-eholdings-provider-list-item
+        to={link}
+        className={cx('item', {
+          'is-selected': active,
+        })}
+        onClick={(e) => {
+          if (onClick) {
+            e.preventDefault();
+            onClick();
+          }
+        }}
+        data-testid="provider-list-item-link"
       >
-        <SelectedLabel
-          isSelected={item.packagesSelected}
-          selectedCount={item.packagesSelected}
-          showSelectedCount
-        />
+        <Headline
+          data-test-eholdings-provider-list-item-name
+          margin="none"
+          size="medium"
+          tag={headingLevel || 'h3'}
+        >
+          {item.name}
+        </Headline>
 
-        <span>
-          <FormattedMessage
-            id="ui-eholdings.label.totalPackages"
-            values={{
-              count: (
-                <span data-test-eholdings-provider-list-item-num-packages-total>
-                  <FormattedNumber value={item.packagesTotal} />
-                </span>
-              )
-            }}
+        <div
+          data-test-eholdings-provider-list-item-selections
+          className={cx('itemMetadata')}
+        >
+          <SelectedLabel
+            isSelected={!!item.packagesSelected}
+            selectedCount={item.packagesSelected}
+            showSelectedCount
           />
-        </span>
-      </div>
-    </InternalLink>
-  );
-}
+
+          <span>
```

Comment:
Do we need this `span`?

Reply:
We can get rid of this `span` only if we also remove `span` around `FormattedNumber` a bit lower. Otherwise it breaks the display of the total packages label. So, should I remove both spans or just leave as it is?

Reply:
Not now, because we need data-attribute here. Thank you for checking and describing it.